docs/src/pages/index.tsx

```diff
@@ -83,9 +104,13 @@ const StyledOcean = styled('div')`
     right: 0;
     overflow: hidden;
     z-index: -1;
+    ${({breakpoint}) => responsiveCSS(breakpoint, [breakpoints.xs, breakpoints.sm])`
+        position:fixed;
+        bottom: -24vw; // Using vw to make it's position more relative to the device width
```

Comment:
Curious what issue this is solving. Also there's no space between the property and value above which means the linter probably hasn't run on the docs site. Can you run that to reformat all the things?

Reply:
Linted. 

Regarding the `vw` unit, this came with a lot of trial and error of various device sizes. I found that using that particular unit gave the most visually appealing (to me) positioning of the waves no matter the device. Having a hard number wasn't working because device widths changed the overall content height. As this was completely subjective measuring I'm open to other options.

Reply:
And now I'm thinking you might have meant the function. It applies the template literal css to multiple breakpoints. I tried a lot of different approaches and landed on the helper function because it was one of the cleaner attempts, and it gave me a chance to explore tagged template literals (admittedly that last one is a selfish reason).

Reply:
I was asking about the `bottom` value, but thanks for clarifying; that makes total sense.